openwisp_radius/admin.py

```diff
@@ -80,19 +76,15 @@ class RadiusCheckAdmin(MultitenantAdminMixin, TimeStampedEditableAdmin):
         'attribute',
         'op',
         'value',
-        'is_active',
-        'valid_until',
         'created',
         'modified',
     ]
     search_fields = ['username', 'value']
     list_filter = [
         DuplicateListFilter,
         ('organization', MultitenantOrgFilter),
-        ExpiredListFilter,
```

Comment:
This filter should also be removed from all filters

Reply:
Will remove it.